tests/integration/optimizers/optimizer_conf.yml

```diff
@@ -0,0 +1,14 @@
+!OptunaOptimizer
+version: 1
+with:
+  flow_runner: !SingleFlowRunner
+    with:
+      flow_yaml: 'tests/integration/optimizers/flow.yml'
```

Comment:
doesn't it capture relative paths? we should maybe use `complete_path` maybe we can achieve similar experience as with other `uses`

Reply:
done